<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>704</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>CalcuX</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 237, 239);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/1.png);}
QPushButton:pressed{border-image: url(:/new/icon/1-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/2.png);}
QPushButton:pressed{border-image: url(:/new/icon/2-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/3.png);}
QPushButton:pressed{border-image: url(:/new/icon/3-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/4.png);}
QPushButton:pressed{border-image: url(:/new/icon/4-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/5.png);}
QPushButton:pressed{border-image: url(:/new/icon/5-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/6.png);}
QPushButton:pressed{border-image: url(:/new/icon/6-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/7.png);}
QPushButton:pressed{border-image: url(:/new/icon/7-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/8.png);}
QPushButton:pressed{border-image: url(:/new/icon/8-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/9.png);}
QPushButton:pressed{border-image: url(:/new/icon/9-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>570</y>
      <width>150</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/0.png);}
QPushButton:pressed{border-image: url(:/new/icon/0-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dot">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/dot.png);}
QPushButton:pressed{border-image: url(:/new/icon/dot-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>495</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/plus.png);}QPushButton:hover{background-image: url(:/new/icon/plus.png);}
QPushButton:pressed{border-image: url(:/new/icon/plus-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sub">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>415</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/sub.png);}QPushButton:hover{background-image: url(:/new/icon/sub.png);}
QPushButton:pressed{border-image: url(:/new/icon/minus-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>575</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/equal.png);}QPushButton:hover{background-image: url(:/new/icon/equal.png);}
QPushButton:pressed{border-image: url(:/new/icon/equal-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mul">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>335</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/mul.png);}QPushButton:hover{background-image: url(:/new/icon/mul.png);}
QPushButton:pressed{border-image: url(:/new/icon/mul-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>65</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/icon/text.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>255</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/div.png);}QPushButton:hover{background-image: url(:/new/icon/div.png);}
QPushButton:pressed{border-image: url(:/new/icon/div-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>106</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>RetroVille NC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 238, 238);
border:none;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exp">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>247</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/EXP.png);}
QPushButton:pressed{border-image: url(:/new/icon/EXP-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>489</x>
      <y>247</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/back.png);}
QPushButton:pressed{border-image: url(:/new/icon/back-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>247</y>
      <width>85</width>
      <height>85</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/AC.png);}
QPushButton:pressed{border-image: url(:/new/icon/ac-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/pow.png);}
QPushButton:pressed{border-image: url(:/new/icon/pow-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_root">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/root.png);}
QPushButton:pressed{border-image: url(:/new/icon/root-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/mod.png);}
QPushButton:pressed{border-image: url(:/new/icon/mod-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/cos.png);}
QPushButton:pressed{border-image: url(:/new/icon/cos-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>410</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/tan.png);}
QPushButton:pressed{border-image: url(:/new/icon/tan-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/sin.png);}
QPushButton:pressed{border-image: url(:/new/icon/sin-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/sin1.png);}
QPushButton:pressed{border-image: url(:/new/icon/sin1-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/cos1.png);}
QPushButton:pressed{border-image: url(:/new/icon/cos1-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_e">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/e.png);}
QPushButton:pressed{border-image: url(:/new/icon/e-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lg">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/lg.png);}
QPushButton:pressed{border-image: url(:/new/icon/lg-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/ln.png);}
QPushButton:pressed{border-image: url(:/new/icon/ln-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pi">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/pi.png);}
QPushButton:pressed{border-image: url(:/new/icon/pi-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>RetroVille NC</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 238, 238);
border:none;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>55</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/icon/text.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rb">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>415</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/rb.png);}
QPushButton:pressed{border-image: url(:/new/icon/rb-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_lb">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>335</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/lb.png);}
QPushButton:pressed{border-image: url(:/new/icon/lb-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_tan_1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/tan1.png);}
QPushButton:pressed{border-image: url(:/new/icon/tan1-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fac">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>570</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/fac.png);}
QPushButton:pressed{border-image: url(:/new/icon/fac-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/FILE.png);}
QPushButton:pressed{border-image: url(:/new/icon/FILE-P.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_HELP">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/help.png);}
QPushButton:pressed{border-image: url(:/new/icon/help-p.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_POSTFIX">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-image: url(:/new/icon/POSTFIX.png);}
QPushButton:pressed{border-image: url(:/new/icon/POSTFIX-P.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pushButton_1</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>pushButton_5</zorder>
   <zorder>pushButton_6</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>pushButton_8</zorder>
   <zorder>pushButton_9</zorder>
   <zorder>pushButton_0</zorder>
   <zorder>pushButton_dot</zorder>
   <zorder>pushButton_plus</zorder>
   <zorder>pushButton_sub</zorder>
   <zorder>pushButton_equal</zorder>
   <zorder>pushButton_mul</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_div</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>pushButton_exp</zorder>
   <zorder>pushButton_back</zorder>
   <zorder>pushButton_clear</zorder>
   <zorder>pushButton_pow</zorder>
   <zorder>pushButton_root</zorder>
   <zorder>pushButton_mod</zorder>
   <zorder>pushButton_cos</zorder>
   <zorder>pushButton_tan</zorder>
   <zorder>pushButton_sin</zorder>
   <zorder>pushButton_sin_1</zorder>
   <zorder>pushButton_cos_1</zorder>
   <zorder>pushButton_e</zorder>
   <zorder>pushButton_lg</zorder>
   <zorder>pushButton_ln</zorder>
   <zorder>pushButton_pi</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_rb</zorder>
   <zorder>pushButton_lb</zorder>
   <zorder>pushButton_tan_1</zorder>
   <zorder>pushButton_fac</zorder>
   <zorder>plainTextEdit_2</zorder>
   <zorder>pushButton_file</zorder>
   <zorder>pushButton_HELP</zorder>
   <zorder>pushButton_POSTFIX</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
